seacatauth/communication/email_smtp.py

```diff
@@ -25,11 +25,13 @@ class SMTPProvider(CommunicationProviderABC):
 		"password": "",
 		"ssl": "no",  # Use TLS/SSL for connection
 		"starttls": "yes",  # Use STARTTLS protocol
+		"_mock": "no",  # Log messages to console instead of sending them
```

Comment:
I suggest to have:

`host=<mocked>`

option instead of introducing a new one